### PR TITLE
Improve new lambda details view with status information

### PR DESCRIPTION
**Pull request by magicmatatjahu**
**Description**

Changes proposed in this pull request:
- Refactor lambda's details header with add status information (like in Instances UI). See picture below.
- Fix subscription handlers for EventTriggers and ServiceBinding
- Extract backend logic (apollo React hooks) to separate hooks - remove ServiceBindingService Provider
- Fix colors of status in lambdas list view
- Fix GreenGuy splitView collapsing on changing tabs.
- Add relevant unit tests

image

**Related issue(s)**
https://github.com/kyma-project/console/issues/1639

### CODE REVIEW

**Comment by michal-hudy** [comment]
Could you remove the `Labels` text and put `pencil` in the same line as labels after them?

**Comment by michal-hudy** [comment]
Also, I noticed that the status in the list view has different formatting than on other lists in UI, please align it with @kwiatekus.

Next question, do we need descriptions for valid statuses (not failed)?

File: core-ui/src/components/Lambdas/helpers/lambdas/logsViewHelpers.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      if (!lambda) {
+        return;
+      }
+
+      const params = createParams(lambda);
+      const title = formatMessage(LOGS_AND_METRICS.LOGS.MODAL.TITLE, {
+        lambdaName: lambda.name,
+      });
+
+      LuigiClient.linkManager()
```
[the remark this answers is not in the document]
Comment (magicmatatjahu) [reply core-ui/src/components/Lambdas/helpers/lambdas/logsViewHelpers.js:33]: `openAs` Modal is leftover for broken functionality and forgot to delete it. Thanks!

File: core-ui/src/components/Lambdas/constants.js [hunk 2]
```diff
@@ -52,47 +146,59 @@ export const GQL_QUERIES = {
...
-    ERROR_MESSAGE: `Error while deleting "{triggerName}" Event Trigger for "{lambdaName}" Lambda: {error}`,
+    ERROR_MESSAGE: `Error while deleting "{triggerName}" Event Trigger for "{lambdaName}" Function: {error}`,
     CONFIRM_MODAL: {
       TITLE: `Remove Event Trigger`,
-      MESSAGE: `Are you sure you want to delete "{triggerName}" Event Trigger for "{lambdaName}" Lambda?`,
+      MESSAGE: `Are you sure you want to delete "{triggerName}" Event Trigger for "{lambdaName}" Function?`,
+    },
+  },
+  CREATE_BINDING_USAGE: {
+    SUCCESS_MESSAGE: `Service Binding of "{serviceInstanceName}" Service Instance was successfully created`,
```
Comment (kazydek) [inline core-ui/src/components/Lambdas/constants.js:193]:
```suggestion
    SUCCESS_MESSAGE: `Service Binding referencing the "{serviceInstanceName}" Service Instance was successfully created`,
```

**Pull request merged**